autobahn/wamp/uri.py

```diff
@@ -282,29 +283,35 @@ def is_exception(self):
 
 
 @public
-def register(uri):
+def register(uri, **options):
     """
```

Comment:
Just to clarify my comment, this would become: `register(uri, options=None)` and "options" should be checked for being a `RegisterOptions` instance (or None) in the decorator itself.